infra/terraform/test-org/org-iam-policy/cloudbuild.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  - name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
+    dir: 'infra/terraform/test-org/org-iam-cleaner'
+    id: 'terraform-init'
+    entrypoint: terraform
+    args: ['init']
+  - name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
```

Comment:
Probably don't need the plan step since we're auto-approving anyways.

Reply:
I decided to keep this in, just so that we can see the changes to catch any bugs. I will remove in a subsequent PR.

Reply:
Oh if this is the onl way to see the changes we should leave it in.